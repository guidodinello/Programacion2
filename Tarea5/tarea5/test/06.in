# Prueba de arregloAAvl
arregloAAvl 1 50
alturaDeAvl
raizAvl
izqAvl
derAvl
enOrdenAvl
arregloAAvl 7 10 20 30 40 50 60 70
buscarEnAvl 40
alturaDeAvl
raizAvl
izqAvl
derAvl
enOrdenAvl
arregloAAvl 8 10 20 30 40 50 60 70 80
buscarEnAvl 40
alturaDeAvl
raizAvl
izqAvl
derAvl
enOrdenAvl
Fin
