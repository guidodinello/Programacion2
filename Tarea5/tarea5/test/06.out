1>#  Prueba de arregloAAvl.
2>
(50,0.00)
3>Altura de avl: 1.
4>50
5>Izquierdo es vacío
6>Derecho es vacío
7>50 
8>
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
9>
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
10>Altura de avl: 3.
11>40
12>20
13>60
14>10 20 30 40 50 60 70 
15>
---(80,0.00)
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
16>
---(80,0.00)
--(70,0.00)
-(60,0.00)
--(50,0.00)
(40,0.00)
--(30,0.00)
-(20,0.00)
--(10,0.00)
17>Altura de avl: 4.
18>40
19>20
20>60
21>10 20 30 40 50 60 70 80 
22>Fin.
